docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -33,7 +33,7 @@ Java runtime environment
 
 Trino requires a 64-bit version of Java 11, with a minimum required version of 11.0.11.
 Earlier patch versions such as 11.0.2 do not work, nor will earlier major versions such as Java 8.
-Newer major versions such as Java 12 or 13 are not supported -- they may work, but are not tested.
+Newer major versions such as Java 12 or 13, including Java 17, are not supported -- they may work, but are not tested.
```

Comment:
I don't think the previous wording could be misinterpreted as not saying that 17 is unsupported.

Reply:
Thats what happened though ... we just need to be more explicit in this specific case ..

Reply:
Alternatively we can change it to say that nothing besides 11 works.. but that would not necessarily be true..